run/pubsub/.gcloudignore

```diff
@@ -0,0 +1,2 @@
+.pytest_cache
```

Comment:
Is this file needed or should it be added to .gitignore?

Reply:
.gcloudignore keeps things from being uploaded to Cloud Build. We do not want local build caches being sent to Cloud Build.
.gitignore keeps things from being commit to the code repository.